science/ngspice/Portfile

```diff
@@ -134,9 +161,16 @@ subport ngspice-lib {
                         --with-readline=${prefix} \
                         --with-ngshared
 
+    # https://trac.macports.org/ticket/70665
+    # https://github.com/macports/macports-ports/commit/e485b0a0c8125cb9d8bcc6241a9c2016514188c8
+    if {${os.platform} eq "darwin" && ${os.major} <= 10} {
+        configure.cppflags-append \
+                                -D__NOEXTENSIONS__
+    }
+
     # Set CFLAGS and LDFLAGS
-    configure.cflags-append -m64 -O2 -Wall -I${prefix}/include/freetype2 -I${prefix}/include
-    configure.ldflags-append -m64 -L${prefix}/lib
+    configure.cflags-append     -O2 -Wall -I${prefix}/include/freetype2 -I${prefix}/include
+    configure.ldflags-append    -L${prefix}/lib
```

Comment:
@bpdegnan Why is this still adding `-O2` or `-I${prefix}/include` or `-L${prefix}/lib`? I already explained in #25041 why we don't want that.

Reply:
(I agree these should not be there, but I avoided to introduce changes beyond required ones.)

Reply:
Let's pull it out - I missed that one too. Freetype has to stay thanks to some weird goings on with whatever the heck is going on with this build system.

Reply:
The build system inserts some silly flags here and there, maybe just patch those out.